setup.py

```diff
@@ -79,6 +79,7 @@
         ],
         "cms.djangoapp": [
             "ace_common = openedx.core.djangoapps.ace_common.apps:AceCommonConfig",
+            "discussion = lms.djangoapps.discussion.apps:DiscussionConfig",
```

Comment:
This looks concerning - isn't this a clear violation of cms depending on lms?

Reply:
My understanding (please correct me if I'm wrong) was that the discussion app's current home is more of a historical artifact, and that the long term goal is consolidating all its common/client pieces into this single app, moving it out of edx-platform altogether, and using the pluggable apps mechanism to do the wiring. I can put back my changes shifting things into common and remove this line, but I thought this was more directionally in line with where we wanted to take discussions and the discussions/edx-platform dependency.

Reply:
I agree that the current location of the discussions all isn’t its long-term place.  But having CMS calling LMS doesn’t follow our current best practices either.

Perhaps adding a comment to that effect would prevent future trespasses as we introduce this ‘broken window’ here?

Reply:
I agree that the current location of the discussions all isn’t its long-term place.  But having CMS calling LMS doesn’t follow our current best practices either.

Perhaps adding a comment to that effect would prevent future trespasses as we introduce this ‘broken window’ here?

Reply:
I agree that the current location of the discussions all isn’t its long-term place.  But having CMS calling LMS doesn’t follow our current best practices either.

Perhaps adding a comment to that effect would prevent future trespasses as we introduce this ‘broken window’ here?